keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -32,7 +32,7 @@ class MemoryCloudDriverCache(
   private val cloudDriver: CloudDriverService
 ) : CloudDriverCache {
 
-  private val securityGroupSummariesById = Caffeine.newBuilder()
+  private val securityGroupSummariesByIdOrName = Caffeine.newBuilder()
```

Comment:
I'm confused why there's only one cache here but it looks like it's only primed with a single key. If I ask for a security group by id, and then the same security group by name, that's 2 hits on CloudDriver, right?

Reply:
Another good catch. Will fix.